libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -218,6 +220,24 @@ export interface ICSPDirective {
     };
 }
 
+/**
+ * @internal
+ */
+export interface ICustomApplicationSetting {
+    id: string;
+    type: string;
+    /**
+     * Name of the application the setting is applied.
+     * Ex: ldmModeler
+     */
+    applicationName: string;
```

Comment:
Is it really string (can it be any application, even custom one)? Or should it be rather union type?

Reply:
yes, it is because the applicationName is defined by the application, it can be any application.